apps/taxes/src/modules/taxjar/maps/taxjar-calculate-taxes-map.ts

```diff
@@ -1,10 +1,26 @@
 import { TaxForOrderRes } from "taxjar/dist/types/returnTypes";
-import { TaxBaseFragment } from "../../../../generated/graphql";
+import { TaxBaseFragment, TaxBaseLineFragment } from "../../../../generated/graphql";
 import { ChannelConfig } from "../../channels-configuration/channels-config";
-import { taxLineResolver } from "../../taxes/tax-line-resolver";
 import { CalculateTaxesResponse } from "../../taxes/tax-provider-webhook";
 import { FetchTaxForOrderArgs } from "../taxjar-client";
 
+function getTaxBaseLineDiscount(
```

Comment:
such functions must be tested

Reply:
It was tested by Maciek, along with other functions. I am yet to move his tests to the codebase, there is a ticket for that.

Reply:
Why can't you just add tests along with functions you add?

Reply:
It's not a function I added, I moved it from another file. The majority of TaxJar integration was written by Maciek, along with this function. I have no problem with adding tests for this particular function if you want, but I am just letting you know that there is a bigger testing effort already planned for this function and the other TaxJar ones.

Reply:
I think its better if we incrementally add tests together with code, since app is already "live". Adding tests "in the future" can be a false promise, because we can always have other things to do